### PR TITLE
EN-1142 Manage authorizations

### PR DESCRIPTION
**Pull request by fedelocci**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

File: src/ui/users/authority/UserAuthorityTable.js [hunk 1]
```diff
@@ -100,7 +105,7 @@ class UserAuthorityTable extends Component {
             <label className="control-label col-sm-2" htmlFor="widgetType">
```
Comment (DMinnai) [inline src/ui/users/authority/UserAuthorityTable.js]: change class col-sm-2 accordingly to <Col xs={12}

File: src/ui/users/authority/UserAuthorityTable.js [hunk 2]
```diff
@@ -116,7 +121,7 @@ class UserAuthorityTable extends Component {
             <label className="control-label col-sm-2" htmlFor="plugin">
```
Comment (DMinnai) [inline src/ui/users/authority/UserAuthorityTable.js]: change class col-sm-2 accordingly to <Col xs={9}

**Comment by entando-service** [comment]
All is well

**Pull request merged**